content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/package.rst

```diff
@@ -0,0 +1,219 @@
+
+.. image:: package/enable-batch.png
+   :align: center
+   :alt: Activate the *Batch Transfers* feature in Inventory > Configuration > Settings.
+
+Then, go to the package form to configure the package intended for batch picking. To do that,
+navigate to :menuselection:`Inventory app --> Products --> Packages`.
+
+On this page, add new packages by clicking :guilabel:`New`, or select an existing package. Doing so
+opens the package form, which contains the following fields:
+
+- :guilabel:`Package Reference`: (required) name of the package
+- :guilabel:`Pack Date`: the date that the package was created
+- :guilabel:`Package Use`: choose :guilabel:`Reusable` for packages used for moving products within
+  the warehouse; :guilabel:`Disposable` for packages used to ship products to customers.
+- :guilabel:`Package Type`: used for :ref:`configuring shipping boxes to ship to the customer
+  <inventory/warehouses_storage/package-type>`.
+
+  .. note::
+     :guilabel:`Package Type` is unnecessary for configuring batch transfer packages.
+
+- :guilabel:`Shipping Weight`: used to input the weight of the package after measuring it on a
+  scale.
+- :guilabel:`Company`: specify a company to make the package available **only** at the selected
+  company. Leave the field blank if the package is available at all companies.
+
+.. image:: package/package.png
+   :align: center
+   :alt: Display package form to create a cluster pack.
+
+Add to batch
+------------
+
+.. important::
+   The following example uses packages in a :doc:`three-step delivery
+   <../../shipping_receiving/daily_operations/delivery_three_steps>` workflow.
+
+   This workflow requires the :guilabel:`Packages`, :guilabel:`Storage Locations`, and
+   :guilabel:`Multi-Step Routes` features to be enabled in :menuselection:`Inventory app -->
+   Configuration --> Settings`. On the warehouse configuration page (:menuselection:`Inventory app
+   --> Configuration --> Warehouses`), the :guilabel:`Outgoing Shipments` option must be set to
+   :guilabel:`Pack goods, send goods in output and then deliver (3 steps)`.
+
+After confirming a sales order, selecting the :guilabel:`Delivery` smart button displays the three
+tasks to complete the delivery.
+
+.. image:: package/three-step.png
+   :align: center
+   :alt: Clicking "Delivery" smart button displays the pick, pack, ship steps.
+
+To create a batch transfer, navigate to :menuselection:`Inventory app --> Operations --> Internal`
+and select the checkboxes next to the pick operations that are to be added to the batch.
+
+After the pick operations are selected, navigate to the :guilabel:`⚙️ Actions` button and select
+:guilabel:`Add to batch` from the list of drop-down options.
+
+.. image:: package/create-batch.png
+   :align: center
+   :alt: Add pickings to batch.
+
+Doing so opens an :guilabel:`Add to batch` pop-up window. Click :guilabel:`Confirm` to finish
+creating the batch.
```

Comment:
is there any benefit to briefly explaining the fields present on the 'Add to batch' pop-up? Maybe even adding a screenshot could be beneficial.

Reply:
I was thinking, since batches aren't the focus on this packages doc, I didn't take a screenshot here or go into much details about the fields. Instead, I linked a related doc in the seealso block right below this paragraph. 

Do you think that gives enough details? I wanted to get your opinion as a non-Inventory expert. If you think the lack of explanation about batches disrupts your understanding of packages and their use case here, I can go into further detail about batches (: